sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -113,25 +126,32 @@ def check_ortho(M, err_msg):
         [[-0.61330704, -0.00443647,  0.78983213],
          [-0.74697144, -0.32172099, -0.58183269],
          [-0.25668686,  0.94682413, -0.19399983]])
-    assert_array_almost_equal(pls_2.x_weights_, x_weights)
+    x_weights_sign_flip = pls_2.x_weights_ / x_weights
```

Comment:
this looks ok but I feel it would be more natural to compute the sign flip and then compare for R == sign_flip * Python for all of them. but your way works, too.